src/models/tickle.rs

```diff
@@ -28,6 +28,7 @@ pub struct AuthStatus {
     pub connected: bool,
     pub message: String,
     pub server_info: ServerInfo,
+    pub fail: String,
```

Comment:
is this a new field?

Reply:
I only see fail on the `auth/status` endpoint output not on the /tickle -iserver - authStatus endpoint.  We could create a second AuthStatus struct if you would prefer that would include that field.